src/components/avatar/AvatarContent.jsx

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import Icon, {types as ICON_TYPE, colors as ICON_COLOR} from '../icons/Icon';
```

Comment:
Will be more messy if you need import icon type from Icon module and then use it in avatar. Now you use avatar with dependencies exported from same file.